scripts/coverage.js

```diff
@@ -33,9 +33,9 @@ function copyFiles(files, destDir) {
 }
 
 const lcovs = glob.sync(globPattern).map((dir) => {
-    const packageName = dir.split('/').pop();
+    const packageName = dir.split(path.sep).join('/').split('/').pop();
```

Comment:
`glob` had to be updated to fix an issue in [rimraf](https://github.com/isaacs/rimraf/issues/306), which was causing `concerto-types` build to fail.

`glob` itself had a breaking change for the `sync` [API](https://github.com/isaacs/node-glob/issues/480) a few releases back because of which coverge reprting was failing for windows environment in the PR build pipeline.

So I had to fix the script.